test/unit/lib/much-rails/call_method_tests.rb

```diff
@@ -0,0 +1,61 @@
+require "assert"
+
+require "much-rails/call_method"
+
+module MuchRails::CallMethodTest
+  class UnitTests < Assert::Context
+    desc "MuchRails::CallMethod"
+    subject { unit_class }
+
+    let(:unit_class) { MuchRails::CallMethod }
+
+    should "include MuchPlugin" do
+      assert_that(subject).includes(MuchPlugin)
+    end
+  end
+
+  class ReceiverTests < UnitTests
+    desc "receiver"
+    subject { receiver_class }
+
+    setup do
+      MuchStub.tap(subject, :new) { |new_object|
```

Comment:
@nweathersbee - Since you are using `Assert` you can use `Assert.stub_tap` instead of using `MuchStub` directly. I think this is what I've been doing but it's been a hard switch luckily I had previous code before `MuchStub` was broken out of `Assert` to remind me.

@kellyredding - Do we want to switch to using `MuchStub` directly? Kinda nit-picky but if we want to use it directly it's probably more correct to bring it in as a direct dependency. I probably prefer using `Assert`s API because I'd rather not bring in the dependency but I'm ok switching to using `MuchStub`.

Reply:
@jcredding I should probably switch to using `Assert.*`here, and elsewhere. If @kellyredding decides he'd rather use `MuchStub`, I'll change back and bring it in as a direct dependency.

Reply:
@jcredding @nweathersbee Ah good catch! I'd prefer to use Assert's API and not be aware of MuchStub (it is now just an implementation detail of Assert.

Sorry, Neil, I know this will be a kinda difficult switch but I think it is worth it.

Reply:
No sweat, all good!